handlers/match.js

```diff
@@ -182,3 +212,17 @@ handlers[CSGO.ECSGOCMsg.k_EMsgGCCStrike15_v2_WatchInfoUsers] = function(message)
   }
   this.emit('watchList', response);
 }
+handlers[CSGO.EMsg.ClientRichPresenceInfo] = function(data) {
```

Comment:
The messages.js file will need to be updated with the new rich presence message IDs.
![](http://i.imgur.com/orMVih6.png)